Caltech-IPAC/firefly
# Fixed Multiple tickets and Upload Panel improvements

#### Fixed: Multiple tickets and other clean up based on discussion

 - IRSA-3249: fixed labels and clean up
 - IRSA-3930: hips color not changing when showing with images
 - IRSA-3930: bias contrast sliders are off when click up
 - IRSA-3932: color check mark and slider are not updating with color slection
 - IRSA-3938: stretch dialog out-of-sync with color table
 - Firefly-714: GREAT file crashing firefly
 - Firefly-713: table upload
 - Firefly-711: title label background
 - Firelfy-640: upload region files
 - Clean up of upload panel in case where there is only one table or image
 - Upload panel: significant clean up and error checking
 - Upload panel: single extension view
 - Upload panel: show if unsupported file without errors
 - Upload panel: support region files
 - Upload Panel: better file size feedback
 - Fixed: HiPS not centering on the entered position
 - Fixed: GREAT file no longer crashes firefly
 - File Analyzer: recognized regions files

#### Tickets
https://jira.ipac.caltech.edu/browse/IRSA-3929
https://jira.ipac.caltech.edu/browse/IRSA-3930
https://jira.ipac.caltech.edu/browse/IRSA-3932
https://jira.ipac.caltech.edu/browse/IRSA-3933
https://jira.ipac.caltech.edu/browse/IRSA-3936
https://jira.ipac.caltech.edu/browse/FIREFLY-711
https://jira.ipac.caltech.edu/browse/FIREFLY-713 - table upload feedback
https://jira.ipac.caltech.edu/browse/firefly-714 -  GREAT crashing firefly issue
https://jira.ipac.caltech.edu/browse/FIREFLY-640


#### Testing

- IRSA Viewer: https://irsa-multi-ticket.irsakudev.ipac.caltech.edu/irsaviewer/
- Firefly: https://fireflydev.ipac.caltech.edu/irsa-multi-ticket/firefly/

- Tickets: Test each ticket above
- Upload panel testing
  - Upload a region file
  - Upload an unsupported file  (try PDF, TAR, PNG)
  - load files from http://web.ipac.caltech.edu/staff/roby/data-products-test/
  - Supported files should gives errors
  - Test improved single product panel (upload single HDU fits, ipac table, single table VOTables, csv, etc)
